tests/google-cloud-python/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN git clone https://github.com/GoogleCloudPlatform/google-cloud-python.git
 WORKDIR google-cloud-python
 
 # Checkout the latest release.
-RUN git checkout $(git describe --tags)
+# TODO: Enable once run_unit_tests.py is released
```

Comment:
Done.  Asking me to program without TODOs is like asking me to program without my left hand.  But I'm always up for a challenge.